pkg/frontend/authenticate_test.go

```diff
@@ -10242,307 +10242,6 @@ func TestDoAlterStage(t *testing.T) {
 
 }
 
-func TestDoCheckFilePath(t *testing.T) {
-	convey.Convey("doCheckFilePath success", t, func() {
-		ctrl := gomock.NewController(t)
-		defer ctrl.Finish()
-
-		ses := newTestSession(t, ctrl)
-		defer ses.Close()
-
-		bh := &backgroundExecTest{}
-		bh.init()
-
-		bhStub := gostub.StubFunc(&NewBackgroundExec, bh)
-		defer bhStub.Reset()
-
-		pu := config.NewParameterUnit(&config.FrontendParameters{}, nil, nil, nil)
-		pu.SV.SetDefaultValues()
-		ctx := context.WithValue(context.TODO(), config.ParameterUnitKey, pu)
-
-		rm, _ := NewRoutineManager(ctx)
-		ses.rm = rm
-
-		tenant := &TenantInfo{
-			Tenant:        sysAccountName,
-			User:          rootName,
-			DefaultRole:   moAdminRoleName,
-			TenantID:      sysAccountID,
-			UserID:        rootID,
-			DefaultRoleID: moAdminRoleID,
-		}
-		ses.SetTenantInfo(tenant)
-
-		cs := &tree.Select{}
-		ses.InitExportConfig(cs.Ep)
-
-		//no result set
-		bh.sql2result["begin;"] = nil
-		bh.sql2result["commit;"] = nil
-		bh.sql2result["rollback;"] = nil
-
-		err := doCheckFilePath(ctx, ses, cs.Ep)
```

Comment:
Due the change of implementation of the doCheckFilePath. So the UT is removed in this PR. But doCheckFilePath needs a simpler UT instead of none.